src/reduced_order/pod_interface.h

```diff
@@ -0,0 +1,30 @@
+#ifndef __POD_BASIS_INTERFACE__
+#define __POD_BASIS_INTERFACE__
+
+#include <deal.II/numerics/vector_tools.h>
+#include <deal.II/lac/la_parallel_vector.h>
+#include <deal.II/lac/trilinos_sparse_matrix.h>
+
+namespace PHiLiP {
+namespace ProperOrthogonalDecomposition {
+
+/// Interface for POD
+template <int dim>
+class POD
```

Comment:
any reason this class isn't called `PODBase` ?

Reply:
No reason, I changed it to PODBase in recent commit.

Reply:
Should also rename the file accordingly if it makes sense

Reply:
Done in recent commit.